superbench/runner/runner.py

```diff
@@ -68,10 +69,16 @@ def __get_mode_command(self, mode, exec_command):
         Return:
             str: Runner command.
         """
-        if mode.name == 'torch.distributed':
+        mode_command = exec_command
+        if mode.name == 'local':
+            mode_command = '{prefix} {command}'.format(
+                prefix=(mode.prefix or '').format(proc_rank=mode.proc_rank, proc_num=mode.proc_num or 1),
```

Comment:
Why prefix need to set proc_num=mode.proc_num or 1?

Reply:
it's a format string